### PR TITLE
Add Traditional Chinese `zh_tw.json`

### PR DESCRIPTION
**Pull request by yichifauzi**
Add Traditional Chinese localization support

### CODE REVIEW

File: src/main/resources/assets/autorelog/lang/zh_tw.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "autorelog.midnightconfig.title": "AutoRelog 《自動重新加入》",
+  "autorelog.midnightconfig.interval": "間隔",
+  "autorelog.midnightconfig.interval.tooltip": "重新連線嘗試之間的間隔時間，單位為秒。",
+  "autorelog.midnightconfig.maxAttempts": "最大嘗試次數",
+  "autorelog.midnightconfig.maxAttempts.tooltip": "自動重新加入應進行的重新連線嘗試最大次數。設為 0 代表無限次嘗試。(請確保伺服器管理員同意此設定!)",
+  "autorelog.midnightconfig.afkDetectionComment": "自動閒置偵測",
+  "autorelog.midnightconfig.afkDelay": "閒置延遲",
+  "autorelog.midnightconfig.afkDelay.tooltip": "自動重新加入在閒置狀態下啟動前的延遲時間，單位為秒。",
+  "autorelog.midnightconfig.mode": "模式",
+  "autorelog.midnightconfig.mode.tooltip": "自動：自動重新連線將在閒置時間超過閒置延遲後自動啟動。 手動：輸入指令 「/autorelog」 來啟動或停用自動重新加入。",
```
Comment (notlin4) [inline src/main/resources/assets/autorelog/lang/zh_tw.json:14]:
```suggestion
  "autorelog.midnightconfig.mode.tooltip": "自動：自動重新連線將在閒置時間超過閒置延遲後自動啟動。手動：輸入指令「/autorelog」來啟動或停用自動重新加入。",
```

File: src/main/resources/assets/autorelog/lang/zh_tw.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "commands.autorelog_activated": "自動重新加入已啟用",
+  "commands.autorelog_deactivated": "自動重新加入已停用",
+  "commands.autorelog_error_singleplayer": "自動重新加入在單人遊戲模式下無效",
+  "autorelog.midnightconfig.title": "AutoRelog 《自動重新加入》",
+  "autorelog.midnightconfig.interval": "間隔",
+  "autorelog.midnightconfig.interval.tooltip": "重新連線嘗試之間的間隔時間，單位為秒。",
+  "autorelog.midnightconfig.maxAttempts": "最大嘗試次數",
+  "autorelog.midnightconfig.maxAttempts.tooltip": "自動重新加入應進行的重新連線嘗試最大次數。設為 0 代表無限次嘗試。(請確保伺服器管理員同意此設定!)",
```
Comment (notlin4) [inline src/main/resources/assets/autorelog/lang/zh_tw.json:9]:
```suggestion
  "autorelog.midnightconfig.maxAttempts.tooltip": "自動重新加入應進行的重新連線嘗試最大次數。設為 0 代表無限次嘗試。（請確保伺服器管理員同意此設定！）",
```

File: src/main/resources/assets/autorelog/lang/zh_tw.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "commands.autorelog_activated": "自動重新加入已啟用",
+  "commands.autorelog_deactivated": "自動重新加入已停用",
+  "commands.autorelog_error_singleplayer": "自動重新加入在單人遊戲模式下無效",
+  "autorelog.midnightconfig.title": "AutoRelog 《自動重新加入》",
```
Comment (notlin4) [inline src/main/resources/assets/autorelog/lang/zh_tw.json:5]:
```suggestion
  "autorelog.midnightconfig.title": "AutoRelog《自動重新加入》",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Comment by Scubakay** [comment]
Added in version 0.1.2+1.21.4!

**Pull request merged**